README.md

```diff
@@ -56,6 +57,15 @@ Translations are handled via json files which are located in `src/i18n` folder a
 | **OPTION** | `2` | Show an alert that can be skipped |
 | **NONE** | `3` | Don't display alert at all |
 
+### AlertOptions configuration
+| Property | Default | Description |
+| --- | --- | --- |
+| **custom** | false | Use custom alerts |
```

Comment:
As stated by @jeremypele, as it's an override of the helper defaults, we shouldn't rely on a boolean to toggle the feature, just use the options values if they exist, otherwise use the fallback. What do you think?